p2p/pex/reactor.go

```diff
@@ -16,22 +17,72 @@ var (
 	_ p2p.Wrapper     = (*protop2p.PexMessage)(nil)
 )
 
+// TODO: Consolidate with params file.
```

Comment:
Can we reference an issue here?

Reply:
Sure. I think this is something we should plan to do alongside the removal of the legacy p2p and legacy pex reactor. Should I add these items to the p2p tracking issue (https://github.com/tendermint/tendermint/issues/5670)

Reply:
#6371

Reply:
Can we reference the issue below the TODO?